src/generators/coreFormat.js

```diff
@@ -21,7 +21,7 @@ const regexps = {
   'json-pointer': `(/(?:${FRAGMENT.replace(']*', '/]*')}|~[01]))+`,
 
   // some types from https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#data-types (?)
-  uuid: '^(?:urn:uuid:)?[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12}$',
+  uuid: '[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12}$',
```

Comment:
@pateketrueke It seems like something has broken this. The new file I can find has the same "problem" we fixed some time ago. Is there anything I am missing? I can't find where the specific change happened. https://github.com/json-schema-faker/json-schema-faker/blob/master/src/lib/generators/coreFormat.js#L24

Reply:
can you check please if the change is present on the develop branch?

Reply:
The develop branch seems to be in line. I really do not understand where this change is coming from

https://github.com/json-schema-faker/json-schema-faker/blob/develop/src/lib/generators/coreFormat.js#L24

Reply:
Interesting, please open a new issue with the affected schema, thank you!

Reply:
#600 created.